server.js

```diff
@@ -9,12 +9,11 @@ var secrets = require('./secrets');
 var axios = require('axios');
 
 var app = express();
-var router = express.Router();
```

Comment:
Please use the app.use('/', express.static(__dirname + '/build')); to serve the static files.

Reply:
Please remove Procfile.